tests/modules/net_firewall_policy/examples/hierarchical.yaml

```diff
@@ -74,7 +74,7 @@ values:
     description: null
     direction: INGRESS
     disabled: false
-    enable_logging: null
+    enable_logging: True
```

Comment:
my OCD is triggered by having both `True` and `true` in the same PR

Reply:
well, `true` for Terraform, `True` in yaml for inventories :) But maybe I did not get your comment...

Reply:
ignore me ;)